index.html

```diff
@@ -448,33 +448,28 @@ <h3><a href="#blog-single.html">Business planning, strategy and execution</a> </
     <div class="top-footer">
       <div class="container my-md-5 my-4">
         <div class="row">
-          <div class="col-lg-4">
+          <div class="col-lg-2 ">
             <div class="footer-logo mb-3">
```

Comment:
Add the class `mx-auto` as well. This will horizontally center the element, so in the CSS you don't need to specify `text-align: center;`
[Read more here](https://getbootstrap.com/docs/4.5/utilities/spacing/#horizontal-centering)

Reply:
where am I adding this? what line?